ikasaneip/module/src/main/java/org/ikasan/module/service/ModuleActivatorDefaultImpl.java

```diff
@@ -60,182 +64,188 @@
 
 /**
  * Simple implementation of the default activation of a module.
- * @author Ikasan Development Team
  *
+ * @author Ikasan Development Team
  */
-public class ModuleActivatorDefaultImpl implements ModuleActivator<Flow>
-{
-    /** logger instance */
+public class ModuleActivatorDefaultImpl implements ModuleActivator<Flow> {
```

Comment:
Why was the whitespacing changed? 

I cant see differences and changes on the files